src/common/InitializationCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
```

Comment:
There's a lot of redundant logic here that can be found in `GlobalReadingsList`. Once this is displayed within that component, this could simply turn into a presentational component.

Also, `InitializationMessage` would probably be a better component name since this isn't using the `Card` common component.

Reply:
@mcflav